distribution/tools/entitlement-agent/build.gradle

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the "Elastic License
+ * 2.0", the "GNU Affero General Public License v3.0 only", and the "Server Side
+ * Public License v 1"; you may not use this file except in compliance with, at
+ * your election, the "Elastic License 2.0", the "GNU Affero General Public
+ * License v3.0 only", or the "Server Side Public License, v 1".
+ */
+
+apply plugin: 'elasticsearch.build'
+
+configurations {
+  entitlementRuntime
+}
+
+dependencies {
+  entitlementRuntime project(":libs:elasticsearch-entitlement-runtime")
+  implementation project(":libs:elasticsearch-entitlement-runtime")
+  testImplementation project(":test:framework")
+}
+
+test {
+  dependsOn(jar)
+  jvmArgs "-javaagent:${jar.archiveFile.get().asFile}"
```

Comment:
Have you got an answer from delivery/found out this works as intended and does not need to be dynamic?

Reply:
No, I have not. Let me follow up on that before merging.

Reply:
Got confirmation from @breskeby that he's happy with this.